app/helpers/users_helper.rb

```diff
@@ -9,6 +9,10 @@ def can_view_drafts?
     defined?(current_user).present? && current_user.present? && UserPolicy.new(current_user, nil).drafts?
   end
 
+  def can_view_reports?
+    defined?(current_user).present? && current_user.present? && UserPolicy.new(current_user, nil).drafts?
```

Comment:
```suggestion
    can_view_drafts?
```

Reply:
I think we should leave this as a new method here for clarities sake (even though they do the same things right now) it seems really odd to screen reports by can_view_drafts otherwise and gives us the space to differentiate them in the future

Reply:
I don't think it would be hard to rejig in the future if necessary. I'll leave it up to you.

Reply:
i don't think Mark was suggesting to remove your new method :)

```
def can_view_reports?
  can_view_drafts?
end
```

Reply:
ahh i see